services/server/server_test.go

```diff
@@ -54,6 +54,10 @@ func TestServer(t *testing.T) {
 		mvmalert.Test(t)
 		mvmalert.On("RequestConfigurationUpdate").Return(nil)
 
+		malertmanager := new(mockAlertmanagerService)
+		malertmanager.Test(t)
+		malertmanager.On("RequestConfigurationUpdate").Return(nil)
```

Comment:
maybe check that `RequestConfigurationUpdate` was called?

Reply:
As far as I remember this already should do that.

Reply:
actually no, you should specify number of calls to check and then call `mock.AssertExpectationsForObjects()`

example:

```
func TestMock(t *testing.T) {
	mvmdb := new(mockPrometheusService)
	mvmdb.Test(t)
	mvmdb.On("RequestConfigurationUpdate").Return(nil).Once()

	mock.AssertExpectationsForObjects(t, mvmdb)
}
```

Reply:
Yeah, you are right, we don't call AssertExpectations here

Reply:
I don't see an easy and simple way to add such checks now.

Reply:
In teardown? We have several places where we do it or do you mean that this method is not called in all places?

Reply:
Yes, it's called not in all subtests